src/animation.cc

```diff
@@ -2787,7 +2787,7 @@ void _object_animate()
 
         Object* object = sad->obj;
 
-        unsigned int time = getTicks();
+        unsigned int time = getMultipliedTicks();
```

Comment:
Patching _only_ the usages from the sfall offset list didn't work for me (ie. the dude wasn't faster).
I found I need this particular usage to be patched. I wonder how the sfall solution works without patching this one.